Sources/Examples/Echo/Model/echo.pb.swift

```diff
@@ -3,7 +3,7 @@
 // Generated by the Swift generator plugin for the protocol buffer compiler.
 // Source: echo.proto
 //
-// For information on using the generated types, please see the documentation:
+// For information on using the generated types, please see the documenation:
```

Comment:
typo, possibly in the original proto file?

Reply:
Not the Proto file but the plugin! I fixed this a couple of days ago: https://github.com/apple/swift-protobuf/pull/902 so we should be able to fix this when they tag the next release.